src/components/Todolist.js

```diff
@@ -1,18 +1,15 @@
 import React, { Component } from "react";
 import { firebase } from "../firebase";
-
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 export class Todolist extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
```

Comment:
You can get rid of the constructor in general. Just define `state` as class attribute :)